src/onboarding/Start.js

```diff
@@ -74,7 +76,13 @@ class Start extends React.Component {
     return (
       <Main style={{ opacity: screenTransitionStyles.opacity }}>
         <Content style={screenTransitionStyles}>
-          <StartContent
+          <BreakPoint to="medium">
+            <Warning>
```

Comment:
Only “aragon.org” should be in bold in the message.

Reply:
No longer part of the copy